site/Dockerfile

```diff
@@ -34,6 +34,7 @@ RUN apt-get update -y && apt-get install curl unzip xz-utils git make -y && \
 
 RUN curl -LO https://storage.googleapis.com/flutter_infra_release/releases/stable/linux/flutter_linux_${FLUTTER_VERSION}-stable.tar.xz && \
     tar -xf flutter_linux_${FLUTTER_VERSION}-stable.tar.xz -C / && \
+    git config --global --add safe.directory /flutter && \
```

Comment:
Container builds started breaking for this PR. The `fatal` error was that `flutter` was being run as root during container builds, and this configuration was recommended as a workaround.